rdflib/namespace.py

```diff
@@ -120,7 +120,7 @@ def __getattr__(self, name):
             return self.term(name)
 
     def __repr__(self):
-        return "Namespace(%s)"%text_type.__repr__(self)
+        return "Namespace(%s)" % text_type.__repr__(self)
```

Comment:
@joernhees there's a lot of code here that looks suspicious to me :) as first pass at this module I just took care of some flake8 style fixes which are ancillary to main purpose of this PR (making exception types raised here more specific). I think we should have a separate effort around standardizing code a bit more (switch to use .format() rather than % for string interpolation, etc. etc.). Probably not in scope for this PR though